lib/application/pages/home/collections/update/update_collection_memos.dart

```diff
@@ -1,8 +1,107 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:layoutr/common_layout.dart';
+import 'package:memo/application/constants/dimensions.dart' as dimens;
+import 'package:memo/application/constants/images.dart' as images;
+import 'package:memo/application/constants/strings.dart' as strings;
+import 'package:memo/application/theme/theme_controller.dart';
 
 class UpdateCollectionMemos extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Container(color: Colors.blue, child: Center(child: Text('Memos')));
+    return _CreateMemoEmptyState(
+      onTap: () {
+        print("teste");
+      },
+    );
   }
 }
+
+/// An empty state CTA to add a new `Memo` to a `Collection`.
+class _CreateMemoEmptyState extends ConsumerWidget {
+  const _CreateMemoEmptyState({this.onTap});
+
+  final VoidCallback? onTap;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final theme = ref.watch(themeController);
+    final textTheme = Theme.of(context).textTheme;
+
+    return Material(
+      child: InkWell(
+        onTap: onTap,
+        child: Ink(
+          decoration: BoxDecoration(
+            border: Border.all(color: theme.neutralSwatch.shade700, width: dimens.genericBorderHeight),
+            borderRadius: dimens.executionsTerminalBorderRadius,
+            color: theme.neutralSwatch.shade800,
+          ),
+          child: Center(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                CustomPaint(
+                  size: const Size(dimens.createMemoCtaSide, dimens.createMemoCtaSide),
+                  painter: _CreateButtonPainter(
+                    backgroundColor: theme.primarySwatch.shade400,
+                    createColor: theme.neutralSwatch.shade800,
+                  ),
+                ),
+                context.verticalBox(Spacing.small),
+                Text(
+                  strings.newMemo.toUpperCase(),
+                  style: textTheme.button?.copyWith(color: theme.primarySwatch.shade400),
+                  textAlign: TextAlign.center,
+                )
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class _CreateButtonPainter extends CustomPainter {
```

Comment:
Why do we have a custom (really custom, given that it uses a painter) for this specific flow?

Reply:
The goal of this painter was to draw the + CTA (See SS below). Doing it with a painter gives us the flexibility to customize the + and the background circle colors. I sent a message to Bruno to see if we can make the + in the middle transparent to it use the background color. This way I think we may be able to colorize the background circle using `Asset.color`.

<details>
<summary>Screenshot</summary>

<img width="119" alt="CleanShot 2021-11-21 at 15 15 31@2x" src="https://user-images.githubusercontent.com/11745745/142774035-768b96ea-2a11-41f3-9bac-3abb7e81a3cf.png">

</details>

Reply:
Replaced by an asset